python/perf-kernels/flash-attention.py

```diff
@@ -209,27 +289,54 @@ def _attn_fwd(
     stride_vz, stride_vh, stride_vk, stride_vn,
     stride_oz, stride_oh, stride_om, stride_on,
     stride_bz, stride_bh, stride_bm, stride_bn,
-    extra_tokens_n,
-    H,
-    seqlen_q,
-    seqlen_k,
+    cu_seqlens_q_tensor, cu_seqlens_k_tensor,
     dropout_p,
     philox_seed,
     philox_offset_base,
     encoded_softmax,
+    max_seqlens_q, max_seqlens_k,
+    VARLEN: tl.constexpr,
+    B: tl.constexpr, HQ: tl.constexpr, HK: tl.constexpr,
     STAGE: tl.constexpr,
     BLOCK_M: tl.constexpr,
     BLOCK_DMODEL: tl.constexpr,
     BLOCK_N: tl.constexpr,
     PRE_LOAD_V: tl.constexpr,
-    NEED_PADDING: tl.constexpr,
     BIAS_TYPE: tl.constexpr,
     ENABLE_DROPOUT: tl.constexpr,
     RETURN_ENCODED_SOFTMAX: tl.constexpr
 ):
+    MQA: tl.constexpr = HQ != HK
```

Comment:
Will become `is_mqa` when `HQ` and `HK` become `headdim_q` and `headdim_k`.

Reply:
Fixed (modulo hq becoming headdim_q - it is number of q heads, not the head dim.)